### PR TITLE
Display my alerts

### PR DESCRIPTION
**Pull request by kryswisnaskas**
**Description of change**
Displays alerts for managers, authors and collaborators. If there are no alerts, an empty area will be displayed with the text "You're all caught up" (per design).

**How to test**
- pull changes
- enter a couple of reports
- note that the reports show up in both sections, the "My activity report alerts" and "Activity reports"
- modify .env file to simulate a different user (one that is not a manager or a collaborator)
- restart the backend server
- note that the reports only show up in the lower table and there is a "You're all caught up" section.
- modify .env file to simulate a user that is either a manager or a collaborator on existing reports
- restart the backend server
- verify that the corresponding reports are displayed in the 

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/181
* https://github.com/HHS/Head-Start-TTADP/issues/191
* https://github.com/HHS/Head-Start-TTADP/issues/192

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: src/services/activityReports.js
Comment (dcmcand) [inline src/services/activityReports.js:257]: Should approved reports be shown on the alerts? It seems like that would get pretty cluttered over time.

File: frontend/src/pages/Landing/MyAlerts.js
Comment (jasalisbury) [inline frontend/src/pages/Landing/MyAlerts.js:47]: Can we use `collaborator.id` here?

File: frontend/src/pages/Landing/MyAlerts.js
Comment (jasalisbury) [inline frontend/src/pages/Landing/MyAlerts.js:58]: If the `/activity-summary` is removed the user will be redirected to the `review` page if the report is not in draft mode (the report is waiting for approval or "Needs Action")

**Review by jasalisbury — APPROVED** [review]
Looks good! The striping from the table causes some of the `<Tag>` elements to not show up very well (see the `status` column of the first row).
Screenshot from 2021-02-16 10-28-13

File: src/services/activityReports.js
Comment (kryswisnaskas) [reply src/services/activityReports.js:257]: True.  No need to have approved reports. Op.ne (not equal) will take care of it.

**Comment by kryswisnaskas** [comment]
> The striping from the table causes some of the <Tag> elements to not show up very well (see the status column of the first row).

Yes, it blends a bit. Double checked the background color. It matches the design, but I'll run this by @arickalewis1

File: src/services/activityReports.js
Comment (kryswisnaskas) [reply src/services/activityReports.js:257]: Hmm...Op.ne (not equal) should take care of not bringing in approved reports. Do you still see 'approved' in the alerts section?

Comment (dcmcand) [reply src/services/activityReports.js:257]: Screenshot from 2021-02-16 13-07-06

This report was filled out and submitted by userid 5 and approved by userid 1. When I log backin as userid 5 it shows up.

**Review by kryswisnaskas — COMMENTED** [review]
@dcmcand Good find! I changed the query condition.

**Review by dcmcand — APPROVED** [review]
Looks good!

File: frontend/src/pages/Landing/MyAlerts.js
Comment (rahearn) [inline frontend/src/pages/Landing/MyAlerts.js:143]: I'm getting this in the console:

`Warning: Failed prop type: Invalid prop `reports` supplied to `MyAlerts`, expected a ReactNode`

File: src/services/activityReports.js
Comment (rahearn) [inline src/services/activityReports.js:251]: I think this should just be submitted, so that when I submit my review the report gets removed from my alerts. Thoughts?

File: src/services/activityReports.js
Comment (rahearn) [inline src/services/activityReports.js:259]: Do any of these status strings exist in a Constant? It'd be good to refer to the constants rather than hard coding in the query.

**Review by rahearn — CHANGES_REQUESTED** [review]

File: src/services/activityReports.js
Comment (rahearn) [reply src/services/activityReports.js:251]: After talking about this with @kryswisnaskas, we're going to leave this for now. If this muddies the list we can create a new story to address.

File: src/services/activityReports.js
Comment (kryswisnaskas) [reply src/services/activityReports.js]: Good catch!
Fixed.

File: frontend/src/pages/Landing/MyAlerts.js
Comment (kryswisnaskas) [reply frontend/src/pages/Landing/MyAlerts.js]: Changed.

**Review by rahearn — APPROVED** [review]

**Pull request merged**
